core/src/test/scala/integration/kafka/server/QuorumTestHarness.scala

```diff
@@ -204,10 +205,19 @@ abstract class QuorumTestHarness extends Logging {
     TestInfoUtils.isShareGroupTest(testInfo)
   }
 
-  def maybeGroupProtocolSpecified(testInfo: TestInfo): Option[GroupProtocol] = {
+  def maybeGroupProtocolSpecified(): Option[GroupProtocol] = {
     TestInfoUtils.maybeGroupProtocolSpecified(testInfo)
   }
 
+  def groupProtocolFromTestParameters() = {
+    val gp = maybeGroupProtocolSpecified()
+
+    if (gp.isEmpty)
+      throw new IllegalStateException("Please specify the group.protocol configuration when creating a KafkaConsumer")
```

Comment:
`maybeGroupProtocolSpecified` is based on the input arguments of test case. Maybe the error message should be "Please specify the groupProtocol=consumer when writing the test"

Reply:
I updated the error message to `Please specify the "groupProtocol" parameter when writing the test`. I didn't use `groupProtocol=consumer` in the error message because the value of `groupProtocol` can be `classic` or `consumer`.

PTAL. Thanks!